settings.ini

```
@@ -3,17 +3,17 @@ autoStart=1
 initialRes=100
 
 [Resolution change]
-minRes=60
-maxRes=500
+minRes=25
+maxRes=200
```

```suggestion
minRes=40
maxRes=400
```
IMO these are too stretched. 40% is the lowest I'd ever go. I'd say we still to prioritize the user being able to see above getting their target FPS (if their GPU usage is that high, they might be re-projecting from CPU too btw)
Not sure what your argument about lowering maxRes is, could you elaborate? 200% is definitely too low, at least for low-res HMDs like HTC Vive and Valve Index. I can still see an increase in Quality in the 300s with an Index.

---

> Not sure what your argument about lowering maxRes is, could you elaborate?

On lower-res HMDs, the default resolution in SteamVR is always a fixed value. On the Quest 2, whether through Oculus Link, Air Link, or even Virtual Desktop, (which means Pico headsets get lumped in too) it is _not._ 

Furthermore, due to compression artifacts even when running at higher bitrates, the effects of supersampling aren't noticeable on Quest 2 until you go to around 140% resolution. Given that most Quest 2 users will optimize their "100%" resolution to be 1x render scale, (and for Oculus Link and Air Link, their Oculus Debug Tool encoder settings as well) having there be such a large gap between potential max resolution and actual expected resolution can cause things to go haywire very quickly.

**Especially** if the game they're playing supports MSAA, and I'm pretty sure every game this app supports either:
-A: Natively supports MSAA
-B: Is too demanding for the resolution numbers to go that high
-C: both A and B

The only game I can think of that COULD take advantage of such numbers on the Quest 2 is Fracked, and Fracked is a UE4 game, which means I doubt this app would work with that game anyway.

---

maxRes could be at 500% or 200% and it wouldn't matter if the user does not have the GPU headroom to achieve it. There's no point in lowering it, is there

---

> maxRes could be at 500% or 200% and it wouldn't matter if the user does not have the GPU headroom to achieve it. There's no point in lowering it, is there

🤔 🤔 🤔 

"if on Quest 2 or Pico, lower this to 100"

---

The thing is that there’s not really any reason to lower it normally at all. If there was a reason, it could be lowered, but there’s none…

---

Also the problem with comments is that I don’t believe .ini files supports them at all

---

> Also the problem with comments is that I don’t believe .ini files supports them at all

oh right i think we should add the settings explanations back into that in-download readme again so that you don't need to pull up the github every time

---

Sure. If we agree that lowering max is useless tho, keep it at 500% 3:<